Description: Added compiler flag generating debug information.
  With the current provided cmake flags, no debug information was generated in
  the compilation. The proper flag is added to either CMAKE_C_FLAGS and
  CMAKE_CXX_FLAGS.
  .
  This patch should be forwarded upstream.
Author: Raúl Sánchez Siles <rasasi78@gmail.com>
Last-Update: 2012-03-15
Origin: vendor, http://git.debian.org/?p=pkg-kde/kde-extras/kvirc.git;a=blob_plain;f=debian/patches/30_upstream_build-g;hb=HEAD
Forwarded: no
---
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -220,8 +220,8 @@
 		ENDIF()
 		IF(CMAKE_COMPILER_IS_GNUCXX)
 			#force gdb options
-			SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -O3 -fomit-frame-pointer -DNDEBUG --no-enforce-eh-specs -pipe --exec-charset=UTF-8 --input-charset=UTF-8 --no-implement-inlines --unit-at-a-time --fast-math")
-			SET(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -O3 -fomit-frame-pointer -DNDEBUG --no-enforce-eh-specs -pipe --exec-charset=UTF-8 --input-charset=UTF-8 --no-implement-inlines --unit-at-a-time --fast-math")
+			SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -O3 -fomit-frame-pointer -DNDEBUG --no-enforce-eh-specs -pipe --exec-charset=UTF-8 --input-charset=UTF-8 --no-implement-inlines --unit-at-a-time --fast-math -ggdb")
+			SET(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -O3 -fomit-frame-pointer -DNDEBUG --no-enforce-eh-specs -pipe --exec-charset=UTF-8 --input-charset=UTF-8 --no-implement-inlines --unit-at-a-time --fast-math -ggdb")
 			INCLUDE(CheckCXXCompilerFlag)
 			CHECK_CXX_COMPILER_FLAG("-fvisibility-inlines-hidden" CXX_HAS_VIH_FLAG)
 			#gcc doesn't support visibility on PE/win32
